core/modules/system/system.theme.inc

```diff
@@ -144,6 +144,8 @@ function theme_system_themes_page($variables) {
 function theme_system_modules_fieldset($variables) {
   $form = $variables['form'];
 
+  $modules_data = system_rebuild_module_data();
```

Comment:
@RakeshPotnuru can we please change this variable name to singular instead of plural? (same as the function name called):
```suggestion
  $module_data = system_rebuild_module_data();
```

Reply:
I think it contains data of different modules?

Reply:
To me 'data' is plural, so $module_data works fine for multiple modules.

Reply:
ok